### PR TITLE
Add analyse details page

### PR DESCRIPTION
**Pull request by Karkavin**

### CODE REVIEW

File: angio-frontend/src/modules/analyse/components/AnalyseDetailsAdditionalInfo.vue [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+        @Prop()
+        public analyse!: Analyse;
+
+        public fullName(name: FullName): string {
+            const patronymic: string = name.patronymic ? name.patronymic : '';
+            return `${name.lastname} ${name.firstname} ${patronymic}`;
+        }
+
+        public isNotEmpty(data?: string): boolean {
```
Comment (nikon-petr) [inline angio-frontend/src/modules/analyse/components/AnalyseDetailsAdditionalInfo.vue]: Move to utility class

File: angio-frontend/src/modules/analyse/components/AnalyseDetailsAdditionalInfo.vue [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                        </v-flex>
+                        <v-flex xs8>
+                            <div class="subheading">
+                                <div>
+                                    <span class="font-weight-medium headline">
+                                        {{ $t('analyse.component.details.additionalInfo.info.title') }}
+                                    </span>
+                                </div>
+                            </div>
+                            <div v-if="analyse.additionalInfo.diagnostician && analyse.additionalInfo.diagnostician.fullName">
```
Comment (nikon-petr) [inline angio-frontend/src/modules/analyse/components/AnalyseDetailsAdditionalInfo.vue]: Move the condition to method

File: angio-frontend/src/modules/analyse/views/AnalyseDetails.vue [hunk 3]
```diff
@@ -43,18 +87,24 @@
...
+                        .then((starredRes) => {
+                            this.analyse = analyseRes.data.data;
+                            this.analyse.starred = starredRes.data.data.starred;
+                            this.starred = starredRes.data.data.starred;
 
-                    PatientApiService.getPatientById(this.analyse.additionalInfo.patientId)
-                        .then((res) => {
-                            this.patient = res.data.data;
-                            this.setDocumentTitle();
+                            PatientApiService.getPatientById(this.analyse.additionalInfo.patientId)
```
Comment (nikon-petr) [inline angio-frontend/src/modules/analyse/views/AnalyseDetails.vue]: Why `PatientApiService.getPatientById(...)` call depends on `AnalyseApiService.getAnalyseStarred(...)`?

File: angio-frontend/src/modules/analyse/components/AnalyseDensityTab.vue [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+                            v-bind:items="densities"
+                            class="elevation-1"
+                            hide-actions
+                    >
+                        <template slot="headerCell" slot-scope="props">
+                            {{ $t(props.header.text) }}
+                        </template>
+                        <template slot="items" slot-scope="props">
+                            <td>{{ props.item.sectorNumber }}</td>
+                            <td class="text-xs-center">{{ Math.round(props.item.density  * 100) / 100 }}</td>
```
[the remark this answers is not in the document]
Comment (Karkavin) [reply angio-frontend/src/modules/analyse/components/AnalyseDensityTab.vue]: Done

File: angio-frontend/src/modules/analyse/components/AnalyseDetailsAdditionalInfo.vue (hunk 1 above)
Comment (Karkavin) [reply angio-frontend/src/modules/analyse/components/AnalyseDetailsAdditionalInfo.vue]: Moved

File: angio-frontend/src/modules/analyse/components/AnalyseDetailsAdditionalInfo.vue (hunk 2 above)
Comment (Karkavin) [reply angio-frontend/src/modules/analyse/components/AnalyseDetailsAdditionalInfo.vue]: Moved

File: angio-frontend/src/modules/analyse/views/AnalyseDetails.vue (hunk 3 above)
Comment (Karkavin) [reply angio-frontend/src/modules/analyse/views/AnalyseDetails.vue]: Fixed

Comment (nikon-petr) [reply angio-frontend/src/modules/analyse/views/AnalyseDetails.vue]: `AnalyseApiService.getAnalyseStarred(...)` and `PatientApiService.getPatientById(...)` depends on `AnalyseApiService.getAnalyseById(...)` only. Not from each other.

**Pull request merged**